app/prepdocs/PrepareDocs/Program.Clients.cs

```diff
@@ -15,11 +15,11 @@ internal static partial class Program
     private static readonly SemaphoreSlim s_searchLock = new(1);
 
     private static Task<BlobContainerClient> GetCorpusBlobContainerClientAsync(AppOptions options) =>
-        GetLazyClientAsync<BlobContainerClient>(options, s_corpusContainerLock, async o =>
+        GetLazyClientAsync<BlobContainerClient>(options, s_corpusContainerLock, static async o =>
```

Comment:
Not pushing back on this change. Just curious as to what adding static does here.

Reply:
It's pure, rather than capturing variables and lifting them to an outer scope.